src/core/chat-api.ts

```diff
@@ -91,9 +100,18 @@ export class ChatApi {
 
   async getMessages(roomId: string, params: GetMessagesQueryParams): Promise<PaginatedResult<Message>> {
     roomId = encodeURIComponent(roomId);
-    return this._makeAuthorizedPaginatedRequest<Message>(`/chat/v2/rooms/${roomId}/messages`, params).then((data) => {
-      return this._recursivePaginateMessages(data);
-    });
+
+    // convert the params into internal format
+    const apiParams: ApiGetMessagesQueryParams = { ...params };
+    if (params.orderBy) {
+      apiParams.direction = params.orderBy === OrderBy.OldestFirst ? 'forwards' : 'backwards';
```

Comment:
if someone is using plain javascript without type hints and puts 'forwards' then this will default to 'backwards' since it's only checking whether the param === 'oldestFirst'. i'd recommend using a switch statement and throwing an error if the param isn't a valid `OrderBy` member